lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
Can we just do a alias for now, so the old rake task name remains until the next major version release?